src/main/java/com/spotify/autoscaler/Autoscaler.java

```diff
@@ -50,7 +53,7 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(Autoscaler.class);
   private static final int SHORT_TIMEOUT = (int) Duration.ofSeconds(10).toMillis();
   private static final int LONG_TIMEOUT = (int) Duration.ofSeconds(60).toMillis();
-  private static final boolean USE_TIMEOUT = true;
+  private static final int THREADS_TIMEOUT = (int) Duration.ofSeconds(5).toMillis();
```

Comment:
The reason I set it to 5 is the following. The overall scheduling in `Application.java` triggers every 5 seconds, so I thought that if there is some sort of contention in the threads created during that process, we should probably time them out before the next iteration starts, that is less than 5 seconds. If we increase this, I would also lean towards increasing the scheduling interval. WDYT?

Reply:
It actually starts 5 seconds _after_ the first run finishes, so not every 5 seconds. So if it takes 30 seconds to complete, it will run at t=0, t=35, t=70, and so on